deploy/complete/helm-chart/mushop/charts/orders/templates/orders-deployment.yaml

```diff
@@ -20,6 +20,9 @@ spec:
         mockmode: {{ .Values.global.mock.service | quote }}
       annotations:
         sidecar.istio.io/rewriteAppHTTPProbers: "true"
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /actuator/prometheus
+        prometheus.io/port: "80"
```

Comment:
The actuator defaults to port 80?

Reply:
Yes - defaults to whatever port the web server is running on - jetty in this case and it runs on 80.